gammapy/datasets/metadata.py

```diff
@@ -0,0 +1,157 @@
+import logging
+from typing import Optional, Union
+import numpy as np
+from astropy.coordinates import SkyCoord
+from pydantic import ValidationError, validator
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+__all__ = ["MapDatasetMetaData"]
+
+
+class MapDatasetMetaData(MetaData):
+    """Metadata containing information about the GTI.
+
+    Parameters
+    ----------
+    creation : `~gammapy.utils.CreatorMetaData`
+        the creation metadata
+    instrument : str
+        the instrument used during observation
+    telescope : str
+        The specific telescope subarray
+    observation_mode : str
+        observing mode
+    pointing : ~astropy.coordinates.SkyCoord
+        Telescope pointing direction
+    obs_ids : int
+        Observation ids stacked in the dataset
+    event_types : int
+        Event types used in analysis
+    optional : dict
```

Comment:
I share your suggestion to not store the individuals GTIs, but I am wondering whether a TSTART and a TSTOP can not be still useful. Any thought?

Reply:
I guess it depends a bit on how this meta information will be used